packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -305,21 +312,21 @@ describeNoCompat("GC data store sweep tests", (getTestObjectProvider) => {
 		itExpects(
 			"Receiving ops for swept datastores fails in client after sweep timeout and summarizing container",
 			[
-				{
-					eventName:
-						"fluid:telemetry:ContainerRuntime:GarbageCollector:SweepReadyObject_Loaded",
```

Comment:
Is this a generic event now and not an error event?

Reply:
No, these won't be logged at all. I mentioned this in the PR description:
The other tests were not testing the inactive / sweep ready state but rather tombstone / deleted state. For these, the container does not need to load in "write" mode. So, for these tests removed the expected inactive / sweep ready errors since they won't happen in loaded containers anymore.

Let me know if it makes sense.